MultiTable/test/RealtimeChangeHandler.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * test/RealtimeChangeHandler.js
+ */
+'use strict';
+const expect = require('chai').expect;
+const RealtimeChangeHandler = require('../src/RealtimeChangeHandler');
+const SubjectGroups = require('../src/SubjectGroups');
+const SubjectGroup = require('../src/SubjectGroup');
+
+describe('./test/RealtimeChangeHandler.js >', () => {
+  const sgArgs = { absolutePath: 'a', name: 'a' };
+
+  it('ok - valid data arg and valid chg arg', () => {
+    const data = new SubjectGroups(sgArgs);
+    const chg = { 'sample.add': { name: 'a|b', aspect: { name: 'b'} } };
+    RealtimeChangeHandler.handle(chg, data);
+    expect(data instanceof SubjectGroups).to.equal.true;
+    expect(data).to.have.property('rootSubject', 'a');
+    expect(data.map).to.have.property('a');
+  });
+
+  describe('handle: validateData >', () => {
+    it('instance of wrong class', () => {
+      expect(() => RealtimeChangeHandler.handle({}, new SubjectGroup('x')))
```

Comment:
is there a reason for using arrow functions here?

Reply:
I was just following the chai docs for expect(...).to.throw(...)

Reply:
ok, I read it. Makes sense.